jerry-core/api/jerry.c

```diff
@@ -33,9 +33,14 @@
 #include "ecma-promise-object.h"
 #include "jcontext.h"
 #include "jerryscript.h"
+#include "jmem.h"
 #include "js-parser.h"
 #include "re-compiler.h"
 
+#define JMEM_ALLOCATOR_INTERNAL
+#include "jmem-allocator-internal.h"
+#undef JMEM_ALLOCATOR_INTERNAL
```

Comment:
Why do we need this include? If something is needed, we should move it to jmem.h.